.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "files.exclude": {
-    "**/*.element.css.ts": true
+    "**/*.element.css.ts": true,
+    "**/*.global.css.ts": true
```

Comment:
New global style option for components. This is used for components that need to use browser specific pseudo selectors that don't work in shadow dom. (range, input icons)